libs/wire-api/src/Wire/API/User.hs

```diff
@@ -1151,3 +1155,31 @@ instance S.ToSchema ListUsersQuery where
           & S.description ?~ "exactly one of qualified_ids or qualified_handles must be provided."
           & S.properties .~ InsOrdHashMap.fromList [("qualified_ids", uids), ("qualified_handles", handles)]
           & S.example ?~ toJSON (ListUsersByIds [Qualified (Id UUID.nil) (Domain "example.com")])
+
+-----------------------------------------------------------------------------
+-- 2nd Factor Auth
+-- todo(leif): roundtrip tests
+-- todo(leif): golden tests
+
+data SecondFactorAuthAction = GenerateScimToken
+  deriving stock (Eq, Show, Generic)
```

Comment:
```suggestion
  deriving stock (Eq, Show, Enum, Bounded, Generic)
```

(if you want.  we can also save some compile time and add these if we need them, not sure what's the best call here.)